src/main/java/org/apache/accumulo/access/Authorizations.java

```diff
@@ -64,7 +65,8 @@ public String toString() {
   }
 
   public static Authorizations of(String... authorizations) {
-    return new Authorizations(Set.of(authorizations));
+    // Use Set.copyOf to remove duplicates if they exist
+    return new Authorizations(Set.copyOf(Arrays.asList(authorizations)));
```

Comment:
I think it would be better to keep the `Set.of()` so it issues an obvious error to the user if they supply duplicates, since duplicates could indicate a user typo or similar error.